### DISCUSSION TITLE
How to view which JS packages get included in the build bundle?

### DISCUSSION THREAD

**Discussion by LukeCSYello** [vercel/next.js#66504] [Help]
### Summary

Hi,

I am using a package to produce a confetti effect, installed by NPM, and then imported as normal. This is in one component, currently (and for the foreseeable future) used on only one page. Obviously, I don't need this Javascript loaded on every page. The chunking and obfuscation Next does during its build process has made it effectively a black box for me to understand whether this JS is being unnecessarily included, and I couldn't see anything on the docs, apart from [this](https://nextjs.org/docs/pages/building-your-application/optimizing/lazy-loading), which would imply to be I would want to wrap my import in dynamic(). Am I understanding correctly and could anyone point me in the right direction with this topic?

Thanks

### Additional information

```js
Using pages router not app router
```

**Comment by SouthLink** [comment]
You can use [@next/bundle-analyzer](https://nextjs.org/docs/app/building-your-application/optimizing/bundle-analyzer)
